### PR TITLE
fix warnings for swift 1.2

### PR DESCRIPTION
**Pull request by mrschroeder**
updated to compile without warnings in swift 1.2

### CODE REVIEW

File: KSTokenView/KSToken.swift [hunk 1]
```diff
@@ -174,8 +174,5 @@ class KSToken : UIControl {
          rectanglePath.stroke()
       }
    }
-   
-   func description() -> String {
```
Comment (arpi0003) [inline KSTokenView/KSToken.swift:178]: I see.  What we could do is `override var description: String { return title }` to keep with the authors intentions.  I don't really care either way

**Comment by mrschroeder** [comment]
@mpillsbury @arpi0003 @joe - have new repo for this and folder links from SNM - you can test the change with the included objc example project

File: KSTokenView/KSToken.swift (hunk 1 above)
Comment (mrschroeder) [inline KSTokenView/KSToken.swift:178]: it's not called anywhere so hard to know their intent

**Comment by mrschroeder** [comment]
@arpi0003 - followed your suggestion

**Comment by arpi0003** [comment]
@mrschroeder looks good.  made one small comment

**Pull request merged**